frontend/packages/ceph-storage-plugin/src/selectors/index.ts

```diff
@@ -32,3 +34,12 @@ export const getCephSC = (scData: K8sResourceKind[]): K8sResourceKind[] =>
       _.get(sc, 'provisioner', '').includes(provisioner),
     );
   });
+
+export const getOCSVersion = (items: FirehoseResult): string => {
+  const itemsData: K8sResourceKind[] = _.get(items, 'data');
+  const operator: K8sResourceKind = _.find(
+    itemsData,
+    (item: K8sResourceKind) => _.get(item, 'spec.name') === OCS_OPERATOR,
```

Comment:
type not required as it's inherited from the `itemsData`.
```suggestion
    (item) => _.get(item, 'spec.name') === OCS_OPERATOR,
```